src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -1269,7 +1272,7 @@ public function count()
     private function formatValue($value): string
     {
         if (\is_object($value)) {
-            return \get_class($value);
+            return $value::class;
```

Comment:
this should not be applied in 5.4 as we performed this migration only in 6.x

Reply:
I reverted all fabbot.io suggestions thanks